www/src/App.js

```diff
@@ -23,7 +27,46 @@ const StyledRoot = styled(Router)`
   }
 `
 
+const FullScreen = styled.div`
+position: absolute;
+width: 100%;
+height: 100%;
+`
+
 function App() {
+  const [translations, setTranslations] = useGlobalSlice('translations')
+  const [language, setLanguage] = useGlobalSlice('language')
+
+  const languageFromLS = localStorage.getItem(config.LANGUAGE_KEY);
+  let fallbackLanguage = config.availableLanguages[0];
+  try {
+    fallbackLanguage = JSON.parse(languageFromLS) || fallbackLanguage;
+  } catch (ignore) {}
+
+  useEffect(() => {
+    if(language) {
+      fetchTranslation(language.value).then(translation => {
```

Comment:
Prefer `async / await` with a `try / catch` block over `Promise.then` and `Promise.catch`